ckanext/canada/i18n/fr/LC_MESSAGES/ckanext-canada.po

```diff
@@ -2028,10 +2030,12 @@ msgid "Ensure that your account is linked to your organization."
 msgstr "Assurez-vous que votre compte est lié à votre organisation."
 
 #: ckanext/canada/templates/internal/help.html:98
+#, fuzzy
 msgid ""
 "If your organization hasn't uploaded data yet for that proactive "
 "disclosure type, your template hasn't been initiated. To access the "
-"Upload button, you'll need to click the <em>\"Get Started\"</em> button."
+"Upload button, you'll need to click on the <em>\"Get Started\"</em> "
+"button."
 msgstr ""
 "Si votre organisation n’a pas encore téléversé de données pour ce type de"
 " divulgation proactive, votre modèle n’a pas été initié. Pour accéder au "
```

Comment:
@RabiaSajjad I think we have a wrong French word in here for the `Upload button` translation. I think currently it is `bouton de téléchargement` which is "Download button", so needs updating to `bouton de téléversement`